app/services/scorecard_detail_service.js

```diff
@@ -160,27 +109,120 @@ const getScorecardDetail = async (scorecardUuid) => {
   return await realm.objects('Scorecard').filtered(`uuid == '${scorecardUuid}'`)[0];
 }
 
-const updateScorecardDownloadStatus = (scorecardUuid) => {
+const _getScorecardDownload = (scorecardUuid) => {
```

Comment:
@limkimsan move this method into scorecard_download_service then include using here